pkg/idp/component_test.go

```diff
@@ -681,5 +652,138 @@ func TestDeleteIdentityProviderMapper(t *testing.T) {
 		err := idpComponent.DeleteIdentityProviderMapper(ctx, realmName, idpAlias, mapperID)
 		assert.Nil(t, err)
 	})
+}
+
+func TestDeleteUser(t *testing.T) {
+	var mocks = createMocks(t)
+	defer mocks.finish()
 
+	var (
+		idpComponent = mocks.createComponent()
+		accessToken  = "TOKEN=="
+		userID       = "user-id"
+		groupName    = "the-group"
+		anyError     = errors.New("any error")
+		ctx          = context.TODO()
+	)
+
+	mocks.logger.EXPECT().Warn(gomock.Any(), gomock.Any()).AnyTimes()
```

Comment:
Is this relevant? I feel like it's the same as not having this line at all

Reply:
It is necessary... another way of ignoring logs is to use log.NewNopLogger() rather than using mock.NewLogger(...)